src/v/cluster/health_monitor_types.cc

```diff
@@ -237,6 +241,7 @@ void adl<cluster::node_health_report>::to(
       r.current_version,
       r.id,
       std::move(r.redpanda_version),
+      r.uptime,
```

Comment:
after the next release we'll have to start bumping the version?

Reply:
exactly, since this wasn't yet released it is safe to add this without version bump